windows/src/test/manual-tests/installer/README.md

```diff
@@ -6,13 +6,13 @@
 2. Use the same username/password as your host machine (so psexec doesn't need credentials)
 3. Configure the machine for automatic login (https://technet.microsoft.com/en-us/library/ee872306.aspx)
 4. Add a world-writeable share (e.g. Temp = C:\temp)
-5. Install the test root certificates from /windows/src/buildutils/certificates into the root store on the VM
+5. Install the test root certificates from /common/windows/delphi/tools/certificates into the root store on the VM
```

Comment:
Not sure if overkill but may need mention about the SC_PWD if the certificates have a password.

Reply:
It's in the windows/src/README.md: 

https://github.com/keymanapp/keyman/pull/6737/files#diff-c462a1982117311b2ab5484cf931ef79488659ef95a96108e1e36e7263480004R67-R71